docs/source/ui/pages/run_management.md

```diff
@@ -92,9 +92,11 @@ Params tab contains a JSON-like visualization of all the tracked params data rel
 
 Metrics tab contains the visualizations of all the metrics tracked for the given run.
 
-**Note:** you can track arbitrary number of runs with lots of steps with Aim!
+When checking a run results, there are some metrics that are looked at first. Instead of scrolling down to find those metrics each time, you can pin them to the top. In the same way you can unpin those metrics.
```

Comment:
What about this?

When exploring run results, some metrics are looked at more often. Pin those metrics to the top of the page to find them easier and faster. Undo pinning by clicking on "unpin" button.